ha_mroonga.cpp

```diff
@@ -14954,56 +14954,123 @@ mrn_io_and_cpu_cost ha_mroonga::scan_time()
 }
 
 #ifdef MRN_ENABLE_WRAPPER_MODE
-double ha_mroonga::wrapper_read_time(uint index, uint ranges, ha_rows rows)
+mrn_io_and_cpu_cost ha_mroonga::wrapper_read_time(uint index,
```

Comment:
Could you define `ha_mroonga::wrapper_read_time()` and `ha_mroonga::wrapper_keyread_time()` instead of sharing this implementation for both of `read_time()` and `keyread_time()`.

Sharing implementation is tricky and implementations of them may be changed in the future.